apps/coscrad-frontend/src/components/term-of-the-day/term-of-the-day.container.tsx

```diff
@@ -0,0 +1,46 @@
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+import { useLoadableTermById } from '../../store/slices/resources';
+import { TermOfTheDayPresenter } from './term-of-the-day.presenter';
+
+export const TermOfTheDayContainer = (): JSX.Element => {
+
+    //get termOfTheDay from config instead of hardwired const termArray
+    const { termOfTheDay } = useContext(ConfigurableContentContext);
+
+    // const termArray: string[] = [
+    //     "9b1deb4d-3b7d-4bad-9bdd-2b0d7b110001",
+    //     "9b1deb4d-3b7d-4bad-9bdd-2b0d7b110002",
+    //     "9b1deb4d-3b7d-4bad-9bdd-2b0d7b110511",
+    //     "9b1deb4d-3b7d-4bad-9bdd-2b0d7b110512",
+    //     "9b1deb4d-3b7d-4bad-9bdd-2b0d7b110513",
+    //     "9b1deb4d-3b7d-4bad-9bdd-2b0d7b110501",
+    //     "9b1deb4d-3b7d-4bad-9bdd-2b0d7b110502"
+    // ];
+
+    // #TODO Change from random to select specific day
+
+    const randomIndex = Math.floor(Math.random() * termOfTheDay.length);
+    const randomTerm = termOfTheDay[randomIndex];
+
```

Comment:
Here's where I would get the date. You might want to write a custom hook like `useDate()`. It seems like the most idiomatic React way of doing it, but feel free to in-line it for now.

Reply:
Created useDate().